lisa/operating_system.py

```diff
@@ -220,6 +244,46 @@ def type_name(cls) -> str:
     def name_pattern(cls) -> Pattern[str]:
         return re.compile(f"^{cls.type_name()}$")
 
+    @property
+    def release_version(self) -> VersionInfo:
+        release_version = self._get_os_version().release
+        if VersionInfo.isvalid(release_version):
+            return VersionInfo.parse(release_version)
+
+        return self._coerce_version(release_version)
+
+    def _coerce_version(self, version: str) -> VersionInfo:
```

Comment:
it looks we can merge the logic, it doesn't need an extra method `_coerce_version`

Reply:
> it looks we can merge the logic, it doesn't need an extra method `_coerce_version`

I think it is cleaner this way and actually thinking now that it might be useful to keep it non-private.